cupy/fft/_callback.pyx

```diff
@@ -193,6 +193,7 @@ cdef inline void _mod_compile(str tempdir, str mod_name, str obj_host) except*:
 # other SMs always use sm_X0's symbols only and do not have their own
 # specializations.
 cdef dict _cc_major_map = {
+    '10': ('100',),
```

Comment:
FWIW, "12" is not listed here because of a known issue: https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cufft-release-12-8